src/cart_total/cart_total.jsx

```diff
@@ -29,13 +33,13 @@ const CartTotal = () => {
                     <p className='item_header_subtotal'>Subtotal</p>
                 </div>
                 <div className='cart_total_products'>
-                    {products.map(product => {
-                        let productNumber = product.number
+                    {selectedProducts.map(product => {
+                        let productNumber = product.quantity
```

Comment:
```suggestion
                        let productQuantity = product.quantity
```
you were so close to doing it right, you had all the pieces... why rename `quantity` into `number`? `quanity` is so much more informative